### ISSUE TITLE
[Filter] Issue with Keyboard Navigation on Tab Out

### ISSUE THREAD

**Issue by georgi-sla** [telerik/kendo-angular#4440]
### Description

Currently, when the user focuses on the Filter using the Tab keyboard key, navigates through the component with the keyboard, moves the focus away from the Filter, and again focuses on it with the Tab, the keyboard navigation does not work.

### Steps To Reproduce

1. Create a Filter component.
2. Focus on it using the Tab key.
3. Navigate through the component.
4. Move the focus away from the Filter using the Tab key.
5. Focus again on the component using the Tab key.
6. Try to navigate the component using the left and right arrows.

Please, check out the following StackBlitz demo: https://stackblitz.com/edit/angular-hvm4cw

### Actual Behavior

After moving the focus away from the Filter and then focusing on it again using the Tab, the keyboard navigation of the component stops working as expected.

### Expected Behavior

The keyboard navigation of the Filter should always work no matter whether the component has already been focused or not.

### Browser

All

### Browser version

latest

**Comment by slavenai** [comment]
Fixed in `v17.1.0`
